src/screens/FeedScreen/Header.js

```diff
@@ -484,26 +484,26 @@ const styles = StyleSheet.create({
     fontFamily: fonts.inter[400],
     fontSize: 14,
     lineHeight: 24,
-    color: COLORS.black
+    color: colors.black
   },
   textComment: {
     fontFamily: fonts.inter[400],
     fontSize: 12,
     lineHeight: 18,
-    color: COLORS.gray8
+    color: colors.gray
   },
   usernameComment: {
     fontFamily: fonts.inter[500],
     fontWeight: '900',
     fontSize: 12,
     lineHeight: 24,
-    color: COLORS.black
+    color: colors.black
   },
   usernameTextComment: {
     fontFamily: fonts.inter[500],
     fontSize: 12,
     lineHeight: 24,
-    color: COLORS.gray8
+    color: colors.gray
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color values have been updated to use the new `colors` import. Ensure that these new color values are equivalent to the old ones and that they match the intended design.
```diff
-    color: COLORS.black
+    color: colors.black

-    color: COLORS.gray8
+    color: colors.gray

-    color: COLORS.black
+    color: colors.black

-    color: COLORS.gray8
+    color: colors.gray
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->